tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -108,7 +106,7 @@ def _get_length_sequences_where(x):
 
 
 def _estimate_friedrich_coefficients(x, m, r):
-    """
+    r"""
```

Comment:
Just out of curiosity: why did you choose this and not to add "\\" as in the other cases?

Reply:
It doesn't seem like this is done in a uniform way throughout the codebase.